addons/product_matrix/static/src/js/section_and_note_widget.js

```diff
@@ -65,15 +65,24 @@ SectionAndNoteFieldOne2Many.include({
         // even if there isn't any.
         // the only solution would be to use a custom event catched on a new controller
         // on the so/po form (as a js_class).
-        this.getParent().trigger_up('field_changed', {
+        this.trigger_up('field_changed', {
             dataPointID: this.dataPointID,
             changes: {
                 grid_product_tmpl_id: {id: productTemplateId}
             },
             viewType: 'form',
             onSuccess: function (result) {
-                // why the hell do we receive a table of 8 classes here? :o
-                self._openMatrixConfigurator(result[0].recordData.grid, productTemplateId, editedCellAttributes);
+                // result = list of widgets
+                // find the one of the grid field
+                // and get the grid information from its recordData.
+                var gridInfo = "";
```

Comment:
You can replace the next few lines with this I think:
`var gridInfo = result.find(r => r.recordData.grid);`

Reply:
Can we use arrow functions in 13.0 ? (or should I target master for my fiximps?)

Reply:
you can use arrow functions (and all ES6 features)

Reply:
Can I remove IE11 from supported browsers in the 13.0 doc then?

Reply:
that would actually be wise... We should actually have done that before

Reply:
:arrow_right: https://github.com/odoo/odoo/pull/40602